### PR TITLE
Makefile handle spaces in path

### PR DESCRIPTION
**Pull request by dagar**
@seanrowens @mrpollo please give this a try.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -98,7 +98,7 @@ else
 	PX4_MAKE_ARGS = -j$(j) --no-print-directory
 endif
 
-SRC_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+SRc_DIR:=$(shell dirname "$(realpath $(lastword $(MAKEFILE_LIST)))")
```
Comment (mrpollo) [inline Makefile]: typo on `SRc_DIR`

Comment (mrpollo) [reply Makefile]: fixing the typo this works 👍

**Comment by mrpollo** [comment]
This still fails if there's a space between chars in a path

**Comment by dagar** [comment]
It fixes the first issue (initial SRC_DIR), but there are many more. Standby...

	 % make list_config_targets           
	[nuttx_]aerocore2_default
	[nuttx_]aerofc-v1_default
	[nuttx_]aerofc-v1_rtps
	[nuttx_]auav-x21_default
	[nuttx_]av-x-v1_default
	[nuttx_]crazyflie_default
	[nuttx_]esc35-v1_default
	[nuttx_]mindpx-v2_default
	[nuttx_]nxphlite-v3_default
	[nuttx_]omnibus-f4sd_default
	[nuttx_]px4cannode-v1_default
	[nuttx_]px4esc-v1_default
	[nuttx_]px4fmu-v2_default
	[nuttx_]px4fmu-v2_lpe
	[nuttx_]px4fmu-v2_test
	[nuttx_]px4fmu-v3_default
	[nuttx_]px4fmu-v3_rtps
	[nuttx_]px4fmu-v3_stackcheck
	[nuttx_]px4fmu-v4_default
	[nuttx_]px4fmu-v4pro_default
	[nuttx_]px4fmu-v4pro_rtps
	[nuttx_]px4fmu-v4_rtps
	[nuttx_]px4fmu-v4_stackcheck
	[nuttx_]px4fmu-v5_default
	[nuttx_]px4fmu-v5_rtps
	[nuttx_]px4fmu-v5_stackcheck
	[nuttx_]px4io-v2_default
	[nuttx_]px4nucleoF767ZI-v1_default
	[nuttx_]px4-same70xplained-v1_default
	[nuttx_]px4-stm32f4discovery_default
	[nuttx_]s2740vc-v1_default
	posix_bbblue_cross
	posix_bbblue_native
	posix_bebop_default
	posix_eagle_default
	posix_eagle_hil
	posix_eagle_legacy
	posix_eagle_muorb
	posix_excelsior_default
	posix_excelsior_legacy
	posix_ocpoc_cross
	posix_ocpoc_ubuntu
	posix_rpi_cross
	posix_rpi_cross_no_shield
	posix_rpi_native
	posix_sitl_broadcast
	posix_sitl_default
	posix_sitl_rtps
	posix_sitl_test
	qurt_eagle_default
	qurt_eagle_hello
	qurt_eagle_hil
	qurt_eagle_legacy
	qurt_eagle_muorb
	qurt_eagle_test
	qurt_eagle_travis
	qurt_excelsior_default
	qurt_excelsior_legacy

**Comment by mrpollo** [comment]
Yes just noticed, good luck!

**Comment by mrpollo** [comment]
@dagar make worked OK with your changes, but failed further down the line on `sitl_run.sh`, this commit fixes jmavsim

**Comment by dagar** [comment]
I'm not sure if there's a better way to do this, but it works for now.

**Pull request merged**